### PR TITLE
mark 1 of the dojo

### PR DESCRIPTION
**Pull request by SethBailey**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -493,6 +514,333 @@ private void FoundGold()
...
+                            maxDamage += 15;
+                            MinDamage += 15;
+                            TypeWriter.WriteLine($"You now deal between {MinDamage} and {maxDamage} damage", TypeWriter.Speed.Talk);
+                            TypeWriter.WriteLine();
+                        }
+                        else
+                        {
+                            TypeWriter.WriteLine("Sorry but you don't have enough Gold");
+                            TypeWriter.WriteLine();
+                        }
```
[the remark this answers is not in the document]
Comment (Stackingit) [reply Program.cs:627]: Perhaps we could also pull out the common code.. the only thing unique is the monster, and the award ?

File: Program.cs [hunk 2]
```diff
@@ -493,6 +514,333 @@ private void FoundGold()
...
+            }
+        }
+
+        private void HasBillBeenKilled()
+        {
+            if (isBillDead == true)
+            {
+                TypeWriter.WriteLine($"{userName} William is dead don't linger on the past", TypeWriter.Speed.Talk);
+                TypeWriter.WriteLine();
+                return;
```
Comment (Stackingit) [inline Program.cs:730]: The return here is not needed

File: Program.cs [hunk 3]
```diff
@@ -493,6 +514,333 @@ private void FoundGold()
...
+                TypeWriter.WriteLine("yes of course I'll try, I know, I know... got to go", TypeWriter.Speed.Talk);
+                TypeWriter.WriteLine(new Text("Hey there my name is ", Colours.Speech, TypeWriter.Speed.Talk),
+                                     new Text("William ", Colours.Bill, TypeWriter.Speed.Talk),
+                                     new Text("and I run the Dojo", Colours.Speech, TypeWriter.Speed.Talk));
+                TypeWriter.WriteLine("And whats your name?", TypeWriter.Speed.Talk);
+                billUserName = Console.ReadLine();
+                hasPlayerSeenBill = true;
+                TypeWriter.WriteLine($"Nice, well {billUserName} before we start you must take an oath OK: yes/no", TypeWriter.Speed.Talk);
+                string Awnser = GetLowerReply();
+                if (Awnser == "yes")
```
Comment (Stackingit) [inline Program.cs:751]: spelling, capital

**Review by Stackingit — CHANGES_REQUESTED** [review]

File: Program.cs [hunk 4]
```diff
@@ -493,6 +514,333 @@ private void FoundGold()
...
+        private void dojoCommonPlace()
+        {
+            bool loop = true;
+            while (loop)
+            {            
+                TypeWriter.WriteLine(new Text("Do you want to train in ", Colours.Speech , TypeWriter.Speed.Talk),
+                                    new Text("(a)attack ", Colours.Attack, TypeWriter.Speed.Talk),
+                                    new Text("or ", Colours.Speech , TypeWriter.Speed.Talk),
+                                    new Text("(d)defence", Colours.Protection, TypeWriter.Speed.Talk));
+                TypeWriter.WriteLine("or (l) to leave");
```
[the remark this answers is not in the document]
Comment (SethBailey) [reply Program.cs:535]: no but now it is fixed

File: Program.cs [hunk 5]
```diff
@@ -493,6 +514,333 @@ private void FoundGold()
...
+
+                TypeWriter.WriteLine(new Text("Do you want to train at level 1, ", Colours.Speech, TypeWriter.Speed.Talk),
+                                     new Text($"({level1Price} Gold) ", Colours.Gold),
+                                     new Text("2 ", Colours.Speech, TypeWriter.Speed.Talk),
+                                     new Text($"({level2Price} Gold) ", Colours.Gold),
+                                     new Text("or ", Colours.Speech, TypeWriter.Speed.Talk),
+                                     new Text("3 ", Colours.Speech, TypeWriter.Speed.Talk),
+                                     new Text($"({level3Price} Gold)", Colours.Gold, TypeWriter.Speed.Talk));
+                TypeWriter.WriteLine("Or (l to leave)");                     
+                string Awnser = GetLowerReply();
```
[the remark this answers is not in the document]
Comment (SethBailey) [reply Program.cs:578]: fixed

File: Program.cs [hunk 6]
```diff
@@ -15,7 +15,7 @@ class TheGame
        public string cottonUserName;
        bool cottonIntro = false;
        int protection;
-       bool canPlayerRun = false;
+       bool canPlayerRun = true;
```
[the remark this answers is not in the document]
Comment (SethBailey) [reply Program.cs:18]: based on what the bool does it's just more descriptive

**Pull request merged**